pkg/model/components/containerd.go

```diff
@@ -44,6 +45,10 @@ func (b *ContainerdOptionsBuilder) BuildOptions(o interface{}) error {
 	containerd := clusterSpec.Containerd
 
 	if clusterSpec.ContainerRuntime == "containerd" {
+		if b.IsKubernetesLT("1.18") {
+			klog.Warningf("kubernetes %s is untested with containerd", clusterSpec.KubernetesVersion)
+		}
+
```

Comment:
Is there any technical reason to add this?

Reply:
Ole Markus requested this change back in 6th Dec.

Reply:
I think we decided later during one of the calls that is not necessary anymore. Should be ok to remove as we are testing all supported versions with containerd.